plugins/export-workbook/src/utils.ts

```diff
@@ -1,3 +1,6 @@
+import { ExportSheetOptions, SheetAddress } from './options'
+import { JSON2SheetOpts, OriginOption } from 'xlsx'
```

Comment:
`SheetAddress` and `OriginOption` are unused types.

```suggestion
import type { ExportSheetOptions } from './options'
import type { JSON2SheetOpts } from 'xlsx'
```